src/app/components/details/release_details.rs

```diff
@@ -57,6 +57,12 @@ glib::wrapper! {
     pub struct ReleaseDetailsWindow(ObjectSubclass<imp::ReleaseDetailsWindow>) @extends gtk::Widget, libadwaita::Window, libadwaita::PreferencesWindow;
 }
 
+impl Default for ReleaseDetailsWindow {
```

Comment:
thanks for taking care of clippy :sweat_smile:

Reply:
:saluting_face: